src/qt/paymentrequestplus.h

```diff
@@ -13,6 +13,17 @@
 #include <QList>
 #include <QString>
 
+// Used in getMerchant() to return state
+enum GetMerchantStatus {
+    PR_NOTINITIALIZED = 0,
```

Comment:
Let me ask again, how do I make this so it's accepted. Is it just removing the init values in the enum? Hand me some code example ;).